README.md

```diff
@@ -1,3 +1,15 @@
 # Signadot Examples
 
 This repository contains examples for Signadot users.
+
```

Comment:
Can you move this into the specific readmes for the examples to make it self-contained? A copy of the same thing in each example directory's readme.